pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -216,12 +217,26 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, releaseI
 		}
 	}
 
+	var rhcosImageBaseURI string
+	if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_IMAGE_BASE_URI_OVERRIDE"); ok && ri != "" {
```

Comment:
I'm sorry but this new env variable cannot be added. These env variables bypass installer input models and we would like to keep them to moving towards zero number.

Can you provide how you expect this variable to be used?

Reply:
Ok I can remove it, I was following the existing conventions, and I was thinking it would be a better alternative to the "hidden" CACHEURL in https://github.com/openshift-metal3/kni-installer/pull/100/commits/c3e0e8ec069c9adb9ade5b914d01cf6461e43e40

The reason this is needed/useful is that for both repeated local testing and CI downloading the large RHCOS image is slow and a waste of bandwidth - also it may be required to avoid hitting installer hard-coded timeouts when using a slow connection that cannot download the images fast enough.

If we remove this variable, I'll need to add a new install-config variable for the baremetal platform that can provide the IP for the CACHEURL (since it's hard-coded in the commit referenced above), would that be a more acceptable approach?